HSTracker/Logging/Parsers/PowerGameStateParser.swift

```diff
@@ -359,8 +359,8 @@ class PowerGameStateParser: LogEventParser {
                 type = matches[0].value
             }
             
-            if matches.count > 2 {
-                cardId = matches[2].value
+            if matches.count > 3 {
+                cardId = matches[3].value
```

Comment:
This feels weird. The regexp used is `".*BLOCK_START.*BlockType=(POWER|TRIGGER).*id=(\\d*).*(cardId=(\\w*)).*Target=(.+).*SubOption=(.+)"`

match[3] is the Target (used below for `actionStartingCardId`).

Reply:
Sorry I re-read your initial comment and now I understand. Indeed, cardId will be assigned `cardId=DAL_060`. If there no specific reason to change, I would just leave it as is.

Reply:
match[3] is cardId itself, and match[4] is the Target.
And please check out another snippet of code in PowerGameStateParser.swift,
`if let currentBlock = currentBlock, let chameleosReveal = eventHandler.chameleosReveal,
                let chameleos = eventHandler.entities[chameleosReveal.0], currentBlock.type == "TRIGGER"
                && (currentBlock.cardId == CardIds.NonCollectible.Neutral.Chameleos_ShiftingEnchantment
                    || currentBlock.cardId == CardIds.Collectible.Priest.Chameleos) && chameleos.has(tag: .shifting) {
                eventHandler.handleChameleosReveal(cardId: chameleosReveal.1)
            }`
So I'm kind of pretty sure cardId should be assigned like DAL_060

Reply:
Right, there are nested groups, I didn't know this was possible but it is. 
https://stackoverflow.com/questions/1313934/how-are-nested-capturing-groups-numbered-in-regular-expressions

Looks good to me.